apps/communications/contacts/js/views/details.js

```diff
@@ -83,15 +84,18 @@ contacts.Details = (function() {
 
     initPullEffect(cover);
 
-    utils.listeners.add({
-      '#toggle-favorite': toggleFavorite,
-      '#details-view-header': [
-        {
-          event: 'action',
-          handler: handleDetailsBack
-        }
-      ],
-      '#edit-contact-button': showEditContact
+    // to avoid race conditions with NFC, we load it before handleDetails
+    LazyLoader.load('/contacts/js/nfc.js', () => {
+      utils.listeners.add({
```

Comment:
I would move this to [1]. @arcturus we already have removed this in #open activity patch! :) 

[1] https://github.com/mozilla-b2g/gaia/pull/30669/files#diff-0712ac8d1e81e3ab4223b4843401ef7bR119